vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -1413,14 +1441,21 @@ def process_chunk(chunk_start, chunk_end, skip_result_store=False):
                 scoring_func=self.scoring_func,
                 e_score_correction_bias=self.e_score_correction_bias,
                 activation=self.activation,
+                ep_size=self.ep_size,
+                ep_rank=self.ep_rank,
+                tp_size=self.tp_size,
+                tp_rank=self.tp_rank,
```

Comment:
The parallel params should be captured by the experts or prepare/finalize classes so they don't need to be added as extra parameters to `apply`